Northstar.Client/mod/scripts/vscripts/ui/menu_ingame.nut

```diff
@@ -85,10 +85,12 @@ void function InitInGameMPMenu()
 	var gameHeader = AddComboButtonHeader( comboStruct, headerIndex, "#MENU_HEADER_GAME" )
 	var leaveButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#LEAVE_MATCH" )
 	Hud_AddEventHandler( leaveButton, UIE_CLICK, OnLeaveButton_Activate )
-	#if DEV
-		var devButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "Dev" )
-		Hud_AddEventHandler( devButton, UIE_CLICK, AdvanceMenuEventHandler( GetMenu( "DevMenu" ) ) )
-	#endif
+	
+	if ( GetConVarBool( "ns_allow_team_change" ) )
+	{
+		var devButton = AddComboButton( comboStruct, headerIndex, buttonIndex++, "#SWITCH_TEAMS" )
```

Comment:
Variable name

Reply:
What about it?

Reply:
It doesn't make sense, its the "switch teams button" but the variable is called `devButton`

Reply:
Oh yeah stupid of my part, i copy pasted from the DEV block but didn't even notice i could change the var name.